src/Uno.UWP/UI/Core/CorePhysicalKeyStatus.cs

```diff
@@ -10,5 +12,19 @@ public partial struct CorePhysicalKeyStatus
 		public bool IsMenuKeyDown;
 		public bool WasKeyDown;
 		public bool IsKeyReleased;
+
+		public bool Equals(CorePhysicalKeyStatus other) =>
+			RepeatCount == other.RepeatCount && ScanCode == other.ScanCode &&
+			IsExtendedKey == other.IsExtendedKey && IsMenuKeyDown == other.IsMenuKeyDown &&
+			WasKeyDown == other.WasKeyDown && IsKeyReleased == other.IsKeyReleased;
+
+		public override bool Equals(object? obj) => obj is CorePhysicalKeyStatus other && Equals(other);
+
+		public override int GetHashCode() => HashCode.Combine(RepeatCount, ScanCode, IsExtendedKey, IsMenuKeyDown,
+			WasKeyDown, IsKeyReleased);
```

Comment:
This will likely fail to build. `HashCode.Combine` is (unfortunately) not available on netstandard2.0, which we do build for.